From 8446b6b8ce1ed06dbd2c14a2e9633c37f7e2a5a7 Mon Sep 17 00:00:00 2001
From: Stephen <stephen@vamrs.com>
Date: Thu, 4 Nov 2021 12:08:06 +0800
Subject: [PATCH 13/17] arm64: dts: radxa zero: remove dai-link-0 node

This patch aims to fix the issue of kernel complains:

    [   19.188742]  fe.dai-link-0: ASoC: no backend DAIs enabled for fe.dai-link-0

Signed-off-by: Stephen <stephen@vamrs.com>
---
 arch/arm64/boot/dts/amlogic/meson-g12a-radxa-zero.dts | 4 ----
 1 file changed, 4 deletions(-)

diff --git a/arch/arm64/boot/dts/amlogic/meson-g12a-radxa-zero.dts b/arch/arm64/boot/dts/amlogic/meson-g12a-radxa-zero.dts
index fb891f726b3f..4b2405cdd7e8 100644
--- a/arch/arm64/boot/dts/amlogic/meson-g12a-radxa-zero.dts
+++ b/arch/arm64/boot/dts/amlogic/meson-g12a-radxa-zero.dts
@@ -166,10 +166,6 @@ sound {
 				       <393216000>;
 		status = "okay";
 
-		dai-link-0 {
-			sound-dai = <&frddr_a>;
-		};
-
 		dai-link-1 {
 			sound-dai = <&frddr_b>;
 		};
-- 
2.33.0

